x-pack/plugins/gis/public/angular/map.html

```diff
@@ -27,6 +27,12 @@
               app-name="'maps'"
               on-submit="updateQueryAndDispatch"
               index-patterns="indexPatterns"
+              show-date-picker="showDatePicker"
+              from="time.from"
+              to="time.to"
+              is-paused="refreshConfig.isPaused"
```

Comment:
I think `date-range-from`, `date-range-to`, and `is-refresh-paused` may be more descriptive.